src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/basic/AntiStatisChamber.java

```diff
@@ -0,0 +1,63 @@
+package com.programmerdan.minecraft.simpleadminhacks.hacks.basic;
+
+import com.destroystokyo.paper.event.player.PlayerLaunchProjectileEvent;
+import com.programmerdan.minecraft.simpleadminhacks.SimpleAdminHacks;
+import com.programmerdan.minecraft.simpleadminhacks.framework.BasicHack;
+import com.programmerdan.minecraft.simpleadminhacks.framework.BasicHackConfig;
+import com.programmerdan.minecraft.simpleadminhacks.framework.autoload.AutoLoad;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.NamespacedKey;
+import org.bukkit.entity.EnderPearl;
+import org.bukkit.entity.Player;
+import org.bukkit.entity.Projectile;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.entity.ProjectileHitEvent;
+import org.bukkit.persistence.PersistentDataContainer;
+import org.bukkit.persistence.PersistentDataType;
+import vg.civcraft.mc.civmodcore.util.ConfigParsing;
+
+public class AntiStatisChamber extends BasicHack {
+
+	@AutoLoad
+	private String pearlLifetime;
+	private NamespacedKey key;
+
+	public AntiStatisChamber(SimpleAdminHacks plugin, BasicHackConfig config) {
+		super(plugin, config);
+		this.key = new NamespacedKey(plugin, "pearl_thrown_time");
+	}
+
+	@EventHandler
+	public void onPlayerThrowPearl(PlayerLaunchProjectileEvent event) {
+		Projectile projectile = event.getProjectile();
+		if (!(projectile instanceof EnderPearl)) {
+			return;
+		}
+		EnderPearl pearl = (EnderPearl) projectile;
+		PersistentDataContainer pdc = pearl.getPersistentDataContainer();
+		pdc.set(key, PersistentDataType.LONG, System.currentTimeMillis());
+	}
+
+	@EventHandler
+	public void onPearlLand(ProjectileHitEvent event) {
```

Comment:
You probably want to add LOWEST priority here to avoid issues with other plugins